datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/EditOwnersModal.tsx

```diff
@@ -331,7 +332,18 @@ export const EditOwnersModal = ({
     function handleBlur() {
         setInputValue('');
     }
+    const [defaultValue, setDefaultValue] = useState<any>();
 
+    // If the modal content is dynamic or the modal is not rendered immediately.
+    // we ned need to use the useEffect hook to ensure that the Select component gets the default value after the modal is mounted.
+    useEffect(() => {
```

Comment:
nice! do we need to do this anywhere else? tags, glossary terms?

Reply:
yes where we want default value we have to do this

Reply:
in previous code we are using default value that's the reason I have added this code but in other place we are not using default value so not added this code there. In other places like glossary, term it is working expected without writing this code. If you want we can remove this code from `EditWonerModal` as well but in future we may required to add default value that's the reason I have kept this code here